packages/cashier/src/components/detail/detail.jsx

```diff
@@ -0,0 +1,63 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+import { Icon, Text } from '@deriv/components';
+import './detail.scss';
+
+const Detail = ({ action, children, className, has_red_color, icon, is_link, title }) => {
```

Comment:
@george-usynin-binary Isn't the component name wired?
Its detail of what? Could you be more specific about it.

Reply:
Also, can it be used as detail for every other component?
If no, could you move it to it's parent folder.

Reply:
sure, will check it

Reply:
so this detail is related to payment agent page, i have removed detail component from shared cashier components and added payment-agent-detail to payment agent pages folder. Also we have payment-agent-detail(s!) component in payment-agent pages which we use in pa-withdrawal-receipt. I will refactor/remove this component in the next part of pa-redesign card (withdrawal flow). so eventually it will be only payment-agent-detail component which we can use everywhere in pa page. hope that it was clear for you, if anything pls text me